docs/http/http-spans.md

```diff
@@ -294,52 +291,49 @@ See the examples for more details about:
 
 ## HTTP server
 
-To understand the attributes defined in this section, it is helpful to read the "Definitions" subsection.
+Read the following section to understand how HTTP server instrumentations should capture server information.
```

Comment:
If this isn't intended to be normative, please remove `should` for any other word.

e.g.

`Read the following section to understand how HTTP server instrumentations are suggested to capture server information.`